### PR TITLE
a user can be created, a user can find his account if he already exis…

### PR DESCRIPTION
**Pull request by KevinRoig**
…ts. Minor css to be improved.
Can be used to work nevertheless.

### CODE REVIEW

**Review by maxime-auneau — APPROVED** [review]

File: src/app/components/user/add-user/add-user.component.scss [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    right: 0;
+    margin: auto;
+    transition: transform .4s ease;
+    cursor: pointer;
+    text-align: center;
+}
+
+.container {
+    width: 50vw;
+    //height: 100%;
```
Comment (bastienapp) [inline src/app/components/user/add-user/add-user.component.scss]: retirer le code mort :)

File: src/app/components/user/view-user/view-user.component.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+export class ViewUserComponent {
+  currentUser: User = {
+    pseudo: '',
+    gender: '',
+  };
+  message = '';
+  sub: any;
+  users: any;
+  allUsers: any;
+  test?: string;
```
Comment (bastienapp) [inline src/app/components/user/view-user/view-user.component.ts]: à quoi servent les variables de test ?

**Review by bastienapp — CHANGES_REQUESTED** [review]

File: src/app/components/user/add-user/add-user.component.scss (hunk 1 above)
Comment (KevinRoig) [reply src/app/components/user/add-user/add-user.component.scss]: Done

File: src/app/components/user/view-user/view-user.component.ts (hunk 2 above)
Comment (KevinRoig) [reply src/app/components/user/view-user/view-user.component.ts]: A rien, c'est un oubli!

**Review by bastienapp — APPROVED** [review]
Top :+1:

**Pull request merged**